### PR TITLE
Added r-rnpn as a package

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by thewchan** [comment]
@conda-forge/staged-recipes @conda-forge/help-r ready for review!

File: recipes/r-rnpn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  recipe-maintainers:
+    - conda-forge/r
+    - thewchan
+
+# The original CRAN metadata for this package was:
+
+# Package: rnpn
+# Title: Interface to the National 'Phenology' Network 'API'
+# Description: Programmatic interface to the Web Service methods provided by the National 'Phenology' Network (<https://usanpn.org/>), which includes data on various life history events that occur at specific times.
+# Version: 0.1.0
```
Comment (mfansler) [inline recipes/r-rnpn/meta.yaml:84]: Why is the attached metadata so ancient? How was the recipe generated? (we may want to document this as a bug)

Comment (thewchan) [reply recipes/r-rnpn/meta.yaml:84]: hmm it's a great question, let me try again

Comment (thewchan) [reply recipes/r-rnpn/meta.yaml:84]: I might have pulled an older version by mistake; I did noticed some discrepancies so I updated manually, let me regenerating again

**Review by mfansler — APPROVED** [review]

**Comment by mfansler** [comment]
Thanks for adding this!
